yelp_package/dockerfiles/uwsgi_exporter-sidecar/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM docker-dev.yelpcorp.com/xenial_yelp
+
+COPY ./uwsgi_exporter/uwsgi_exporter /bin/uwsgi_exporter
+CMD /bin/uwsgi_exporter --stats.uri http://127.0.0.1:8889/ --collect.cores false --web.listen-address :9117
```

Comment:
Is that stats.uri the URI the exporter hits on the service to get its metrics? In that case, I thought the stats port could be set by the service - is there a place we're always overriding it to 8889?